src/zope/app/renderer/__init__.py

```diff
@@ -13,20 +13,23 @@
 ##############################################################################
 """Plain Text Renderer Classes
 
-$Id$
 """
 from zope.component.interfaces import IFactory
-from zope.interface import implements, directlyProvides, Declaration
-from interfaces import ISource, IHTMLRenderer
+from zope.interface import implementer, directlyProvides, Declaration
+from zope.app.renderer.interfaces import ISource, IHTMLRenderer # pylint:disable=unused-import
```

Comment:
That's my best guess. I didn't bother to check if this package itself broke if they were removed because the interesting breakages would of course be elsewhere.